package-managers/nuget.md

```diff
@@ -103,7 +103,7 @@ We will need the URL for the feed we want to connect to. Learn more about [deter
 To register our credentials in the `NuGet.config` file we use the following commands:

```

Comment:
"we use the following commands"--> we can/will/... use the following commands
(misses a verb)